apps/Gruntfile.js

```diff
@@ -174,6 +174,12 @@ describe('entry tests', () => {
           src: ['**'],
           dest: 'build/package/media/skins/craft'
         },
+        {
+          expand: true,
+          cwd: 'node_modules/@code-dot-org/ml-activities/dist/assets',
+          src: ['**'],
+          dest: 'build/package/media/skins/fish'
+        },
```

Comment:
You can see this is very similar to how we handle moving assets from the Minecraft package (above)